README.md

```diff
@@ -1096,9 +1096,9 @@ Advanced Robotics, 2020.
 AAAI Conf. Artificial Intelligence, 2021.
 [PDF](https://ojs.aaai.org/index.php/AAAI/article/view/16171/15978), [Video](https://youtu.be/ow2TsNMo_T4), [Project page](https://sites.google.com/site/gachoncvip/projects/deep-event-based-stereo-matching)
 - <a name="Wang21iros"></a>Wang, Z., Pan, L., Ng, Y., Zhuang, Z., Mahony., R.,   
-*[Stereo Hybrid Event-Frame (SHEF) Cameras for 3D Perception](https://arxiv.org/abs/2110.04988)*,  
+*[Stereo Hybrid Event-Frame (SHEF) Cameras for 3D Perception](https://ieeexplore.ieee.org/abstract/document/9636312)*,  
```

Comment:
Better use the doi link: https://doi.org/10.1109/IROS51168.2021.9636312

Reply:
Agreed and updated. Thanks.